Modules/TemperatureSensor/SeasonSubtract.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Transactions;
+
+namespace sendInfo
+{
+    class SeasonSubtract

```

Comment:
Please decouple the FFT with the seasonality processing. These two logical actions are not necessarily always correlated